packages/arb-token-bridge-ui/tests/e2e/specs/depositNativeToken.cy.ts

```diff
@@ -61,10 +61,6 @@ describe('Deposit native token', () => {
       ...txData
     })
 
-    cy.findTransactionInTransactionHistory({
-      duration: depositTime,
-      ...txData
-    })
```

Comment:
Why was it removed?

Reply:
it isn't needed because it's already on L59-62,
when i was fixing e2es i was trying things and saw this, deleted it and it still worked so kept it deleted as it's super minor